integrationtest/mapper/pojo/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/BridgeIT.java

```diff
@@ -31,22 +34,153 @@
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.DocumentId;
 import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
 import org.hibernate.search.mapper.pojo.model.PojoElement;
+import org.hibernate.search.mapper.pojo.model.PojoModelElementAccessor;
 import org.hibernate.search.util.SearchException;
 import org.hibernate.search.util.impl.integrationtest.common.FailureReportUtils;
 import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
 import org.hibernate.search.util.impl.test.SubTest;
+import org.hibernate.search.util.impl.test.annotation.TestForIssue;
 
 import org.junit.Rule;
 import org.junit.Test;
 
 public class BridgeIT {
 
+	private static final String INDEX_NAME = "IndexName";
+
 	@Rule
 	public BackendMock backendMock = new BackendMock( "stubBackend" );
 
 	@Rule
 	public JavaBeanMappingSetupHelper setupHelper = new JavaBeanMappingSetupHelper( MethodHandles.lookup() );
 
+	/**
+	 * Basic test checking that a "normal" custom type bridge will work as expected.
+	 */
+	@Test
+	@TestForIssue(jiraKey = {"HSEARCH-2055", "HSEARCH-2641"})
+	public void typeBridge() {
+		@Indexed(index = INDEX_NAME)
+		class IndexedEntity {
+			Integer id;
+			String stringProperty;
+			@DocumentId
+			public Integer getId() {
+				return id;
+			}
+			public String getStringProperty() {
+				return stringProperty;
+			}
+		}
+
+		backendMock.expectSchema( INDEX_NAME, b ->
+				b.field( "someField", String.class, b2 -> {
+					b2.analyzerName( "myAnalyzer" ); // For HSEARCH-2641
```

Comment:
@gsmet Here, I check that we forward the information to the backend "you should use this analyzer for this field". Whether the backend actually complies with this instruction is another problem, and we already test that in the backend TCK (`org.hibernate.search.integrationtest.backend.tck.analysis.AnalysisCustomIT` in particular).